project.clj

```diff
@@ -1,23 +1,71 @@
+                   [:email "stepan.burlakov@firebolt.io"]]
+                  [:developer
+                   [:id "ptiurin"]
+                   [:name "Petro Tiurin"]
+                   [:email "petro.tiurin@firebolt.io"]]
+                  ])
 
   :dependencies
   [[io.firebolt/firebolt-jdbc "3.1.0"]]
 
-  :repositories [["snapshots" {:sign-releases false
-                               :url "https://repo.repsy.io/mvn/firebolt/maven-snapshots"
-                               :username :env/REPSY_USER
-                               :password :env/REPSY_PASSWORD}]
-                 ["releases" {:sign-releases false
-                              :url "https://repo.repsy.io/mvn/firebolt/maven"
-                              :username :env/REPSY_USER
-                              :password :env/REPSY_PASSWORD}]
+  :repositories [["releases" {:sign-releases true
+                               :url "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+                               :username :env/MAVEN_REPO_USERNAME
+                               :password :env/MAVEN_REPO_PASSWORD}]
+                 ["snapshots" {:sign-releases true
+                              :url "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+                              :username :env/MAVEN_REPO_USERNAME
+                              :password :env/MAVEN_REPO_PASSWORD}]
                  ["project" "file:repo"]]
 
-  :aliases
-  {"test"       ["with-profile" "test"]}
+  :signing {:gpg-key ~(System/getenv "SIGN_KEY_ID")}
+
+  :plugins [[lein-pprint "1.3.2"]
+            [lein-shell "0.5.0"]]
+
+  :aliases {"file-name" ["with-profile" "uberjar" "pprint" "--no-pretty" "--" ":uberjar-name"]
+            "project-version" ["pprint" "--no-pretty" "--" ":version"]
+            "get-metabase-core" ["do" ["shell" "bash" "-c"
+                                       "TMP_DIR=\\$(mktemp -d) && \\
+                                       wget -nv https://downloads.metabase.com/\\$METABASE_VERSION/metabase.jar -O \\$TMP_DIR/metabase.jar && \\
+                                       mkdir -p repo && \\
+                                       mvn deploy:deploy-file -Durl=file:repo -DgroupId=com.firebolt -DartifactId=metabase-core -Dversion=1.40 -Dpackaging=jar -Dfile=\\$TMP_DIR/metabase.jar"]]}
+
+  :pom-plugins [[org.apache.maven.plugins/maven-source-plugin "3.2.1"
+                 ;; this section is optional, values have the same syntax as pom-addition
+                 {:executions ([:execution [:id "attach-sources"]
+                                [:goals ([:goal "jar"])]
+                                [:phase "deploy"]])}]
+                [org.apache.maven.plugins/maven-javadoc-plugin "3.3.1"
+                 {:executions ([:execution [:id "attach-javadocs"]
+                                [:goals ([:goal "jar"])]
+                                [:phase "deploy"]])}]]
+
+  ;; Fix issue with Azure <-> Maven comunication
```

Comment:
Let's mention this is for GH Actions and if you can find that link with the explanation it would be helpful to paste here.